src/core/util/autofocus.js

```diff
@@ -90,18 +90,37 @@ angular.module('material.core')
  * </div>
  * </hljs>
  **/
-function MdAutofocusDirective() {
+function MdAutofocusDirective($parse) {
   return {
     restrict: 'A',
-    link: postLink
+    link: {
+      pre: preLink
+    }
   };
-}
 
-function postLink(scope, element, attrs) {
-  var attr = attrs.mdAutoFocus || attrs.mdAutofocus || attrs.mdSidenavFocus;
+  function preLink(scope, element, attr) {
+    var attrExp = attr.mdAutoFocus || attr.mdAutofocus || attr.mdSidenavFocus;
```

Comment:
Yes, but this is exactly the same. The mapping would be `md-autofocus` -> `mdAutofocus` and `md-auto-focus` -> `mdAutoFocus`.